src/agent/src/command_handler_utils.cpp

```diff
@@ -15,15 +78,23 @@ DispatchCommand(module_command::CommandEntry commandEntry,
 
     LogInfo("Dispatching command {}({})", commandEntry.Command, commandEntry.Module);
 
-    const auto result = co_await module->ExecuteCommand(commandEntry.Command);
+    const auto timeout = std::chrono::minutes(60);
```

Comment:
Maybe we should analyze if we need to make the timeout more flexible, for example by configuration or even that it can come as data from the api. But I think that may be for another issue.

Reply:
Yes, totally, we have many timers with hardcoded values atm. We need to address them with some customizable configuration.